README.md

```diff
@@ -104,20 +161,26 @@ The following section contains the grammar for the T-REX format in [EBNF](https:
 [^2]: Whether a text is a valid `T-REX` can easily be tested using the [EBNF Evaluator](https://mdkrajnak.github.io/ebnftest/)
 
 ``` ebnf
-trex         = segment , { "+" , segment };
-segment      = key, "$", typedvalue ;
+
+trex         = segment , { "+" , segment } ;
+segment      = valuesegment | tablesegment ;
+valuesegment = key, "$", type, ":", value ;
+tablesegment = tablekey, "$$", tableheader, "::", tablerow, { "::", tablerow } ;
+
+tableheader    = key, "$" , type, {":", key,"$", type } ;
+tablerow       = value, { ":", value } ;
+
+tablekey    = alphanumeric, {punctuation | alphanumeric} ;
 key          = alphanumeric, {punctuation | alphanumeric} ;
-typedvalue   = (numericunit, tvseparator, numericvalue) 
-| (texttype, tvseparator, textvalue)
-| (booltype, tvseparator, boolvalue)
-| (datetype, tvseparator, datevalue) 
-| (binarytype, tvseparator, binaryvalue) (* Should only be used if no combination of segments of other types can adequately represent your data. *)
-| (error, tvseparator, errorvalue) (* Reserved for error handling, e.g. error codes in case the actual value of the segment is unavailable. *);
+type         = numericunit | texttype | extendedtexttype | booltype | datetype | binarytype | error;
+value        = numericvalue | textvalue | boolvalue | datevalue | binaryvalue | errorvalue;
 
 numericunit  = alphanumeric, alphanumeric, [alphanumeric]; (* Unit of Measure Common Code as defined by UN/CEFACT in REC 20 *)
 numericvalue = decimal | scientific ;
 texttype     = "T.A";
 textvalue    = { punctuation | alphanumeric };
+extendedtexttype     = "T.T";
```

Comment:
@apinilabs-manuel  do you have a better suggestion to name this. I'm not convinced by this, but didn't have a better idea